winml/test/api/raw/winml_microsoft.h

```diff
@@ -172,7 +172,7 @@ inline HRESULT GetActivationFactory(
     void** factory) noexcept
 {
     // Fallback to OS binary if the redistributable is not present!
-    auto library = LoadLibraryW(MachineLearningDll);
+    auto library = LoadLibraryExW(MachineLearningDll, nullptr, 0);
```

Comment:
What's the reason for changing to use LoadLibraryEx? dwFlags parameter is set to 0 and according to the documentation: if dwFlags is zero, LoadLibraryEx behaves identically to LoadLibrary.

https://docs.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-loadlibraryexw#remarks

Reply:
LoadLibrary isn't available on Win 8.1